src/main/java/org/gradle/profiler/InvocationSettings.java

```diff
@@ -165,6 +168,10 @@ public File getStudioInstallDir() {
         return studioInstallDir;
     }
 
+    public File getStudioSandboxDir() {
+        return studioSandboxDir;
+    }
```

Comment:
We check this for `null`, so either that should be removed, or this should be marked as `@Nullable` or better yet, turned into an `Optional<File>`.

Reply:
Used optional.